src/nxt_sockaddr.c

```diff
@@ -692,8 +612,11 @@ nxt_sockaddr_unix_parse(nxt_mp_t *mp, nxt_str_t *addr)
      *   are covered by the specified length of the address structure.
      *   (Null bytes in the name have no special significance.)
      */
-    if (path[0] == '@') {
+    switch (path[0]) {
+    case '@':
         path[0] = '\0';
+        /* fall through */
+    case '\0':
         socklen--;
     }
```

Comment:
Nice use of switch/case.

The only thing I'd point out is that the socket name is stored in the config as '_unix:\u0000magic_'. Not saying that's a problem, just pointing it out...

Reply:
> Nice use of switch/case.

It looks soo nice! =)

> 
> The only thing I'd point out is that the socket name is stored in the config as '_unix:\u0000magic_'. Not saying that's a problem, just pointing it out...

I checked that JSON allows that: <https://www.rfc-editor.org/rfc/rfc7159#section-7>

Reply:
The commit title is `Fixed support for abstract Unix sockets.`, but the changelog is `supporting abstract UNIX sockets.`.
Is it ok?

Reply:
The first commit just fixes the small bugs in the old code that already intended to support abstract sockets. That's why the commit message says "Fixing ...". However, since we have never officially supported abstract sockets, I used "Added support ..." in the changes file.